src/staticanalysis/bot/static_analysis_bot/report/phabricator.py

```diff
@@ -48,20 +49,17 @@ def publish(self, issues, revision):
         # and avoid publishing a non related patch from an anlyzer partly activated (allowed paths)
         issues = list(filter(lambda i: i.is_publishable() and i.ANALYZER in self.analyzers, issues))
         analyzers_available = set(i.ANALYZER for i in issues).intersection(self.analyzers)
-        patches = {
-            analyzer: url
-            for analyzer, url in revision.improvement_patches.items()
-            if analyzer in analyzers_available
-        }
+        patches = list(filter(lambda p: p.analyzer in analyzers_available, revision.improvement_patches))
 
         if issues:
 
             # First publish inlines as drafts
             inlines = list(filter(None, [
                 self.comment_inline(revision, issue, existing_comments)
                 for issue in issues
+                if issue.ANALYZER != CLANG_FORMAT
```

Comment:
Maybe we should have a new function in the issue which says whether it can have inline comments or not, although probably clang format is going to be the only one in the foreseeable future.